### ISSUE TITLE
Safari Compatibility Issue - Cannot Submit Questions

### ISSUE THREAD

**Issue by CalebAlbers** [KeystoneIT/Ama#57]
Currently questions cannot be submitted via Safari (9.0.3). When the New Question button is pressed, a text box/form is shown, but it cannot be clicked on or typed within.

Looking into the issue a bit more reveals the following in console log:

``` js
TypeError: null is not an object (evaluating 'b.parentNode.getAttribute')
```

I haven't looked into the code, but I'm willing to guess that the Javascript functions associated with the question form are being executed prior to the DOM elements being created.

![screen shot 2016-05-27 at 9 27 21 am](https://cloud.githubusercontent.com/assets/7110138/15611109/628cf6f2-23ee-11e6-8376-6fbb221d6731.png)

**Comment by meyerben** [comment]
Josh found a fix to this bug.

Flags currently work by changing the background of the question to the same color as the text.
There were a few lines in the CSS file to prevent the user from highlighting the text and seeing the original message.

`* {
    -webkit-user-select: none;
    -moz-user-select: none;
    -ms-user-select: none;
}`

Removing those 5 lines fixed the problem for us. This requires some minor tweaking to change how flagging comments works.

**Comment by jsiems** [comment]
fixed #61
